query-engine/core/src/query_graph_builder/read/many.rs

```diff
@@ -1,8 +1,21 @@
 use super::*;
-use crate::{query_document::ParsedField, ManyRecordsQuery, ReadQuery};
+use crate::{query_document::ParsedField, ManyRecordsQuery, QueryOptions, ReadQuery, THROW_ON_EMPTY};
 use prisma_models::ModelRef;
 
 pub fn find_many(field: ParsedField, model: ModelRef) -> QueryGraphBuilderResult<ReadQuery> {
+    find_many_with_options(field, model, QueryOptions::none())
+}
+
+pub fn find_many_or_throw(field: ParsedField, model: ModelRef) -> QueryGraphBuilderResult<ReadQuery> {
+    find_many_with_options(field, model, THROW_ON_EMPTY.to_vec().into())
+}
+
+#[inline]
+fn find_many_with_options(
+    field: ParsedField,
+    model: ModelRef,
+    options: QueryOptions,
+) -> QueryGraphBuilderResult<ReadQuery> {
```

Comment:
This is the old implementation, which now receives QueryOptions. Previous `find_many` keeps the signature and passes no options `QueryOptions::none`, while `find_many_or_throw` builds a  query with the `THROW_ON_EMPTY` option